authors/apps/notifications/models.py

```diff
@@ -0,0 +1,105 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from django.core.mail import EmailMultiAlternatives
+from django.conf import settings
+
+from django.db import models
+from authors.apps.favorites. models import Favorites
+from authors.apps.comments.models import Comment
+from authors.apps.authentication.models import User
+from authors.apps.articles.models import Article
+from authors.apps.profiles.models import Follow
+
+
+class UserNotification(models.Model):
+    """Class for notifications."""
+
+    reciever = models.ForeignKey(User, to_field='username', on_delete=models.CASCADE)
+    message = models.CharField(max_length=255, null=True)
+    link = models.CharField(max_length=255, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
@3Nakajugo Could we an attribute to this model `read` which is initially a boolean `false. Which can be updated to true when the user reads the notification.

Reply:
okay. let me work on it